src/main/java/org/bankofspring/service/BankOfSpringService.java

```diff
@@ -0,0 +1,23 @@
+package org.bankofspring.service;
+/*
+ * the service interface for the bank, at this stage we support credit and debit
+ * also we have an BankOperationValidator
+ */
+import org.bankofspring.model.Account;
+import org.bankofspring.model.User;
+import org.bankofspring.validator.BankOperationValidator;
+
+public interface BankOfSpringService {
```

Comment:
I still don't understand the point of interfaces for things with only one implementation.